server/game/cards/attachments/03/needle.js

```diff
@@ -0,0 +1,32 @@
+const DrawCard = require('../../../drawcard.js');
+
+class Needle extends DrawCard {
+    setupCardAbilities(ability) {
+        this.whileAttached({
+            effect: ability.effects.modifyStrength(2)
+        });
+
+        this.interrupt({
+            when: {
+                onSacrificed: (e, player, card) => card === this.parent
+            },
+            handler: (context) => {
+                context.skipHandler();
+                this.game.addMessage('{0} uses {1} to return {2} to their hand instead of discard', this.controller, this, this.parent);
```

Comment:
can you change the text here to be 'instead of their discard pile', feels a little awkward atm